feature-libs/customer-ticketing/components/customer-ticketing-create/customer-ticketing-create-dialog/customer-ticketing-create-dialog.component.html

```diff
@@ -52,7 +54,12 @@
             {{ 'customerTicketing.associateTo' | cxTranslate }}
           </span>
           <select class="form-control">
-            <option></option>
+            <option
+              *ngFor="let objects of ticketAssociatedObjects"
```

Comment:
```suggestion
              *ngFor="let associatedObject of ticketAssociatedObjects"
```
should be in singular